modules/home/src/lib/latest-news/latest-news.component.html

```diff
@@ -0,0 +1,50 @@
+<body>
+  <h2 class="p-10px">Últimas Notícias</h2>
+  <section class="container p-10px">
+    <div class="main">
+      <img
+        src="main-notice.svg"
+        alt="imagem noticia principal"
```

Comment:
```suggestion
        alt="notícia principal"
```

Trocar o `alt` para que o Sonar não relate bug de acessibilidade